unity-renderer/Assets/Scripts/MainScripts/DCL/Analytics/ExploreV2Analytics/ExploreV2Analytics.cs

```diff
@@ -98,9 +98,9 @@ public void SendPlaceTeleport(string placeId, string placeName, Vector2Int coord
         public void SendClickOnPlaceInfo(string placeId, string placeName)
         {
             Dictionary<string, string> data = new Dictionary<string, string>();
-            data.Add("event_id", placeId);
-            data.Add("event_name", placeName);
+            data.Add("place_id", placeId);
+            data.Add("place_name", placeName);
```

Comment:
good catch, Santi. I changed it because I was thinking that it was just classical problem of copy-pasted method that occur and I decided to correct it. You see, name of the method contains word `Place` and not word `Event`, from which I concluded possible error here. Should I revert it back?
By the way, Analytics has also duplicated function, removing which would easily highlight this issue very fast
![image](https://user-images.githubusercontent.com/35366872/212043313-9fd35950-9d40-4b09-86f0-0db2d2a90a2f.png)

Reply:
so the last function `SendCLickOnPlaceInfo `contained `data.Add("event_id")` and `data.Add("event_name")` which I corrected to `place_id `and `place_name`